docs/guides/insar_product_guide.md

```diff
@@ -99,9 +99,13 @@ To analyze deformation caused by a single discrete event, such as an earthquake,
 
 ### Processing Options 
 
-!!! important "New Processing Options Coming Soon" 
+!!! important "New Processing Options Released, Water Masking Coming Soon!" 
 
-    The only InSAR processing options currently available in Vertex are the resolution (number of looks), and whether to include the look vector maps and the line-of-sight displacement map. The other options listed below are in the process of being implemented and will be available soon. 
+    Several new InSAR processing options have been released. 
+
+    In addition to the original options to set the resolution (number of looks), and whether to include the look vector maps and the line-of-sight displacement map, users can now include GeoTIFF outputs for the **wrapped phase**, the **local incidence angle**, and a copy of the **DEM** used for processing. 
+
+    The water masking option listed below is in the process of being implemented and will be available soon. 
```

Comment:
```suggestion
    Users can now include GeoTIFF outputs for the **wrapped phase**, the **local incidence angle**, and a copy of the **DEM** used for processing. This is in addition to the original options to set the resolution (number of looks), and whether to include the look vector maps and the line-of-sight displacement map.

    The water masking option is under development and will be available soon. 
```

What you have looks great, I'll approve it and feel free to move forward as-is.  This is my attempt to shorten the overall message and to put the new options front and center; feel free to incorporate all/any/none of it.